docs/csharp/fundamentals/program-structure/main-command-line.md

```diff
@@ -205,6 +210,10 @@ To compile and run the application from a command prompt, follow these steps:
 
     :::code language="csharp" source="./snippets/main-command-line/Factorial.cs":::
 
+At the beginning of the `Main` method the program tests if input arguments were not supplied comparing length of `args` argument to `0` and displays the help if no argument are found.
+<br/>If arguments are provided (`args.Length` is greater than 0) program tries to convert the input arguments to numbers. This will throw an exception if the argument is not a number.
+<br/>After factorial is calculated (stored in `result` variable of type `long`) the verbose result is printed depending on the `result` variable.
```

Comment:
These need to be indented for the markdown linter to pass:

```suggestion
    At the beginning of the `Main` method the program tests if input arguments were not supplied comparing length of `args` argument to `0` and displays the help if no argument are found.<br/>
    If arguments are provided (`args.Length` is greater than 0) program tries to convert the input arguments to numbers. This will throw an exception if the argument is not a number.<br/>
    After factorial is calculated (stored in `result` variable of type `long`) the verbose result is printed depending on the `result` variable.
```